Slim/Schema/Contributor.pm

```diff
@@ -83,6 +85,20 @@ sub contributorRoles {
 	return @contributorRoles;
 }
 
+sub defaultContributorRoles {
+	return grep { __PACKAGE__->typeToRole($_) < MIN_CUSTOM_ROLE_ID } contributorRoles();
+}
+
+sub unifiedArtistsListRoles {
+	my @roles = ( 'ARTIST', 'TRACKARTIST', 'ALBUMARTIST' );
+
```

Comment:
A bunch of simple helper methods. Are they any faster than previous code? No. Is it cleaner code? I'd say so. Plus it's centralized in one place to avoid redundancy.